newIDE/app/src/ProjectManager/index.js

```diff
@@ -192,41 +479,91 @@ const ProjectManager = React.memo(
       projectPropertiesDialogInitialTab,
       setProjectPropertiesDialogInitialTab,
     ] = React.useState('properties');
+    const openProjectProperties = React.useCallback(() => {
+      setProjectPropertiesDialogOpen(true);
+      setProjectPropertiesDialogInitialTab('properties');
+    }, []);
+    const openProjectLoadingScreen = React.useCallback(() => {
+      setProjectPropertiesDialogOpen(true);
+      setProjectPropertiesDialogInitialTab('loading-screen');
+    }, []);
+    const onProjectPropertiesApplied = React.useCallback(
+      (options: { newName?: string }) => {
+        if (unsavedChanges) {
+          unsavedChanges.triggerUnsavedChanges();
+        }
+
+        if (options.newName) {
+          onChangeProjectName(options.newName);
+        }
+        setProjectPropertiesDialogOpen(false);
+      },
+      [unsavedChanges, onChangeProjectName]
+    );
+
+    const [openGameDetails, setOpenGameDetails] = React.useState<boolean>(
+      false
+    );
+    const projectUuid = project.getProjectUuid();
+    const { games, fetchGames } = useGamesList();
+    const { profile } = React.useContext(AuthenticatedUserContext);
+    const userId = profile ? profile.id : null;
+    React.useEffect(
+      () => {
+        fetchGames();
+      },
+      [fetchGames, userId]
+    );
+    const gameMatchingProjectUuid = games
+      ? games.find(game => game.id === projectUuid)
+      : null;
+    const onOpenGamesDashboardDialog = React.useCallback(
+      () => setOpenGameDetails(true),
+      []
+    );
+
     const [
       projectVariablesEditorOpen,
       setProjectVariablesEditorOpen,
     ] = React.useState(false);
+    const openProjectVariables = React.useCallback(() => {
+      setProjectVariablesEditorOpen(true);
+    }, []);
+
+    const [editedPropertiesLayout, setEditedPropertiesLayout] = React.useState(
+      null
+    );
+    const [editedVariablesLayout, setEditedVariablesLayout] = React.useState(
+      null
+    );
+    const onOpenLayoutProperties = React.useCallback((layout: ?gdLayout) => {
+      setEditedPropertiesLayout(layout);
+    }, []);
+    const onOpenLayoutVariables = React.useCallback((layout: ?gdLayout) => {
+      setEditedVariablesLayout(layout);
+    }, []);
+
     const [
       extensionsSearchDialogOpen,
       setExtensionsSearchDialogOpen,
     ] = React.useState(false);
+    const openSearchExtensionDialog = React.useCallback(() => {
+      setExtensionsSearchDialogOpen(true);
+    }, []);
```

Comment:
I'm not sure if it's really useful to create a callback for this state variable setting functions.